src/components/hooks/usePerson.js

```diff
@@ -14,7 +14,12 @@ export default function usePerson(personID) {
 
   useEffect(() => {
     const onError = (error) => {
-      throw error;
+      setPerson({
+        displayName: ' ',
```

Comment:
Could we pass alt text to say something like "Error: this is the error message"? I know it wasn't defined in the JIRA, but might be beneficial. Not a blocker though.

Reply:
I think the log should be good enough.

Reply:
I like the alt idea, something like `Unknown user`